README.md

```diff
@@ -1,14 +1,144 @@
 # Sparkle Web App
 
-Codebase for the Sparkle Platform, brought to you by [Sparkle](https://sparklespaces.com/).
+Codebase for the Sparkle platform, brought to you by [Sparkle](https://sparklespaces.com/), a platform for the most immersive & interactive virtual events.
 
-## Getting started
 
-The instructions below assume you already have a Firebase project setup and configured appropriately.
+### Getting started
 
-**Note**: If you're interested in setting up the project to run in your own environment but you're not sure how to go about it, feel free to [open an issue](https://github.com/sparkletown/sparkle/issues/new) asking for assistance.
+What you will need:
 
-### Frontend
+- A free Firebase account
+- Your favorite command-line interface (e.g. Terminal or zsh on a Mac)
+- Optional: accounts with Stripe, Twillio
+- A little fairy dust
+
+---
+<!-- section 1 -->
+
+## Part One: Firebase Project Setup
+
+### Step 1: Create New Firebase Project
```

Comment:
@whoaitsaimz This is WAY too detailed/verbose to have in the README.

Please make a file in `docs/` and add these instructions there.

Reply:
Will do

Reply:
@whoaitsaimz i'm going to add some details on docs/, let's chat when you're available and see how we can merge our changes